pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-content.adoc

```diff
@@ -1,15 +1,15 @@
 [id="importing-(ContentTerm}_{context}"]
 
-= Importing {ContentTerm}
+= Importing content to Pantheon
```

Comment:
Attribute for both content and Pantheon. We decided to use "ContentTerm", although personally, I'm not sure why "content" isn't appropriate. To be safe, change it back to {ContentTerm}.

@adahms Can you help job my memory why we use {ContentTerm} here instead of "content"?

Reply:
Yes, sure. It's changed. I'm also curious why we use the ContentTerm attribute.